### DISCUSSION TITLE
The borders to show up following the wiki is wrong, where use specified layout

### DISCUSSION THREAD

**Discussion by og900aero** [sxyazi/yazi#442] [1. Q&A] [answered]
What system are you running Yazi on?
Debian 12

What terminal are you running Yazi in?
Alacritty 0.11.0

Yazi version
yazi 0.1.5 (1d1a512 2023-12-07)

Describe the bug

Following the [Tips](https://yazi-rs.github.io/docs/tips#full-border) section of the wiki, I am trying to get borders to show up in yazi. But the left side is wrong. The problem of source is the custom layout:
I use the following setting:

```
[manager]
layout         = [ 0, 2, 2 ]
```

See the screenshot for the problem:
![screenshot_1702074201](https://github.com/sxyazi/yazi/assets/49487376/fbf20bcf-a1b9-4ad3-9076-c4cc05c8e542)

The left frame is broken so that if the first window is disabled with a value of 0.

To Reproduce
Follow the steps on the wiki.

**Comment by sxyazi** [comment, marked as answer]
Try change `ui.Padding.y(1)` to `ui.Padding(1, 0, 1, 1)` for the `Current` folder:

```lua
-- Current
Folder:render(chunks[2]:padding(ui.Padding(1, 0, 1, 1)), { kind = Folder.CURRENT }),
```

**Reply by og900aero** [reply]
Thanks, that's basically good. It's only slightly visible that he actually misses the front window too, because the line hangs over there.

![screenshot_1702118410](https://github.com/sxyazi/yazi/assets/49487376/f47b1d7f-106d-403a-806c-d8fcd1d61961)

**Reply by sxyazi** [reply]
You need to replace `┬` and `┴` at the code with `╭` and `╰`.

**Comment by og900aero** [comment]
> You need to replace `┬` and `┴` at the code with `╭` and `╰`.

Uhh, thats hard for me, never mind.

**Reply by sxyazi** [reply]
Just change:

```lua
		bar("┬", chunks[1].right - 1, chunks[1].y),
		bar("┴", chunks[1].right - 1, chunks[1].bottom - 1),
		bar("┬", chunks[2].right, chunks[2].y),
		bar("┴", chunks[2].right, chunks[1].bottom - 1),
```

to

```lua
		bar("╭", chunks[1].right - 1, chunks[1].y),
		bar("╰", chunks[1].right - 1, chunks[1].bottom - 1),
		bar("╭", chunks[2].right, chunks[2].y),
		bar("╰", chunks[2].right, chunks[1].bottom - 1),
```

should work.

**Reply by og900aero** [reply]
Oh my God, I'm stupid, don't be angry...I found where to make changes.
